test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -207,14 +199,19 @@ var _ = Describe("Azure Container Cluster using the Kubernetes Orchestrator", fu
 					Expect(err).NotTo(HaveOccurred())
 					Expect(pass).To(BeTrue())
 				}
+
+				err = nginxDeploy.Delete()
+				Expect(err).NotTo(HaveOccurred())
+				err = s.Delete()
+				Expect(err).NotTo(HaveOccurred())
 			} else {
 				Skip("No linux agent was provisioned for this Cluster Definition")
 			}
 		})
 	})
 
 	Describe("with a windows agent pool", func() {
-		It("should be able to deploy an iis webserver", func() {
+		/*It("should be able to deploy an iis webserver", func() {
```

Comment:
can we just delete this instead of commenting out?  or if you intend to re-enable someday, add a TODO with the conditions of when this should be uncommented.

Reply:
Will add a TODO, thanks